po/de.po

```diff
@@ -1888,10 +1888,10 @@ msgid ""
 "writing, the latest stable Rust release has these version numbers:"
 msgstr ""
 "Wenn Sie mit dem Code auf Ihrem eigenen System experimentieren möchten, sollten Sie\n"
-"zuerst Rust zu installieren. Befolgen Sie dazu die [Anweisungen in dem Rust\n"
+"zuerst Rust installieren. Befolgen Sie dazu die [Anweisungen in dem Rust\n"
```

Comment:
We decided at some point that we want to use the informal "Du" in the training and not "Sie". Please change here and in the rest of the translation, thanks!